Sources/GRPCProtobufCodeGen/ProtobufCodeGenParser.swift

```diff
@@ -22,9 +22,19 @@ import struct GRPCCodeGen.CodeGenerationRequest
 
 /// Parses a ``FileDescriptor`` object into a ``CodeGenerationRequest`` object.
 internal struct ProtobufCodeGenParser {
-  internal init() {}
-  internal func parse(input: FileDescriptor) throws -> CodeGenerationRequest {
-    var header = input.header
+  let input: FileDescriptor
+  let namer: SwiftProtobufNamer
+
+  internal init(input: FileDescriptor) {
+    self.input = input
+    self.namer = SwiftProtobufNamer(
+      currentFile: input,
+      protoFileToModuleMappings: ProtoFileToModuleMappings()
+    )
+  }
```

Comment:
Sorry, my last comment wasn't clear enough: I want us to pass the `SwiftProtobufNamer` in `init`. We _do_ need the mappings as it turns out, they come from the options (`options.protoToModuleMappings`).